commands/deploy.go

```diff
@@ -160,7 +161,7 @@ func (c *DeployCmd) Run() (err error) {
 			},
 		},
 	}
-	err = api.ApplicationEnvironmentModuleUpdate(c.AccountID, c.AppID, c.Environment, "nodejs/.section-external-source.json", ups)
+	err = api.ApplicationEnvironmentModuleUpdate(c.AccountID, c.AppID, c.Environment, c.AppPath + "/.section-external-source.json", ups)
```

Comment:
[path.Join()](https://golang.org/pkg/path/#Join) would be better than string concatenation here for the case where the user has included the trailing slash in their argument value.